generator/uscore/templates/unit_tests/authorization_unit_test.rb.erb

```diff
@@ -0,0 +1,42 @@
+describe 'unauthorized search test' do
```

Comment:
It would be nice to have a comment at the top indicating that this test is autogenerated.  Also a note that any changes may be overwritten when regenerating tests (I assume that would happen).